staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -319,6 +320,23 @@ func (c *celAdmissionController) Validate(
 					continue
 				}
 			}
+			var namespace *v1.Namespace
+			namespaceName := a.GetNamespace()
+
+			// Special case, the namespace object has the namespace of itself (maybe a bug).
+			// unset it if the incoming object is a namespace
```

Comment:
cc @alexzielenski for visibility since https://github.com/kubernetes/kubernetes/pull/119215 is loosely related